src/js/utils/ui.js

```diff
@@ -241,25 +249,18 @@ const eventRegisters = {
         initInteractionListeners(ui);
     },
     click(ui) {
-        initInteractionListeners(ui);
+        initSelectListeners(ui);
     },
     tap(ui) {
-        if (OS.iOS && OS.version.major < 11) {
-            const body = document.body;
-            if (body) {
-                // When controls are disabled iOS 10 does not dispatch media element touchstart/end events without this line
-                body.ontouchstart = body.ontouchstart || function() {};
```

Comment:
Just a question. Is this something we might still need to handle for ios 10 or is it no longer needed?

Reply:
Noting that this was in the register for `tap`, we no longer need this line. The only case where we were listening to touchstart/touchend events with `controls: false` would be dragging the floating player (doesn't register `tap`) and tapping on the video itself (now handled by click so we don't need the touchstart/end events). With that in mind, we can now safely remove these lines.

cc @robwalch in case I'm missing something here.

Reply:
Note: I've also confirmed no difference in manual testing.